src/components/browse/components/SearchResultTable.js

```diff
@@ -393,12 +393,16 @@ class LoadMoreAsYouScroll extends React.Component {
         return styles;
     }
 
-    static getElementHeight(openDetailPanes, rowHeight, children, openRowHeight){
```

Comment:
No logic has changed here, just refactored a little bit for the logic to be more explicit/clear.